content/en/logs/explorer/saved_views.md

```diff
@@ -21,7 +21,7 @@ Troubleshooting is highly contextual, and Saved Views enable you and your teamma
 
 Technically, a Saved View keeps track of:
 
-- A [search query][3]
+- A [search query][3] along with its timerange. Note that the Saved View is meant to track live timeranges (past hour, past week...) and fixed timeranges are converted as such on save. 
```

Comment:
Since we're adding a period to this first bullet point and making it a complete sentence, we need to do the same to the following bullet points for consistency in style.